lib/auth/iotSas.ts

```diff
@@ -3,6 +3,7 @@
 
 import { SasTokenProvider } from "./sas";
 import { TokenInfo } from "./token";
+import { Buffer } from "buffer";
```

Comment:
What is Buffer being used for?

Reply:
In sas.ts, Buffer is used only in a type position for a parameter to _createToken. In iotSas, `Buffer.from` is used in getToken.

Reply:
So, now both in Nodejs and browser, this `buffer` package will be used?

Reply:
But `buffer` is not part of the `dependencies` in package.json ...

Reply:
Yes both Nodejs and browsers use buffer. Unfortunately rhea depends on buffer. It should be a dev dependency (not needed unless you're doing a browser build).

Reply:
> It should be a dev dependency (not needed unless you're doing a browser build).

When event hubs and service bus end up shipping browser support, then `buffer` would no longer be just a dev dependency...

Are we planning to then add "buffer" as a direct dependency in the event hubs and service bus packages?

Reply:
You're probably right that buffer should be a real dep. In this PR, I ended up doing all the work for amqp-common's dependencies, but that doesn't have to be the case if package.json browser field is set properly and all the necessary deps are regular deps.

The unfortunate downside is that you end up installing those packages even when you're not building for web, but it's probably better that things "just work".

Reply:
Latest version has all the browser build deps moved into a real dependency.